test/WebSites/MvcTagHelpersWebSite/Models/Person.cs

```diff
@@ -28,6 +30,7 @@ public string Password
         }
 
         [EnumDataType(typeof(Gender))]
+        [UIHint("GenderUsingTagHelpers")]
```

Comment:
already have a .cshtml file containing `@Html.EditorFor(m => m.Gender)`.  this hint works with the renamed template file to ensure that template is still found.